plugins/ruby-foreman-puppet/debian/changelog

```diff
@@ -1,3 +1,9 @@
+ ruby-foreman-puppet (3.0.6-1) stable; urgency=low
```

Comment:
this failed to build because there is an extra space here ;)

https://community.theforeman.org/t/foreman-packaging-deb-3-2-release-27-failed/28380

Reply:
cc @pcreech